PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalVaultEditAuthRequest.kt

```diff
@@ -1,22 +0,0 @@
-package com.braintreepayments.api.paypal.vaultedit
-
-import com.braintreepayments.api.core.ExperimentalBetaApi
-
-/**
- * A request used to launch the continuation of the PayPal Edit Vault flow.
- */
-@ExperimentalBetaApi
-sealed class PayPalVaultEditAuthRequest {
```

Comment:
I'm slightly confused why we are removing `PayPalVaulEditAuthRequest` and `PayPalVaultEditAuthRequestParams` since they seem to have the correct names and setup required (can modify the other classes to match this, but these look how it's expected I think)